src/Uno.UI.RuntimeTests/Tests/Windows_UI_Xaml_Controls/Given_ListViewBase.cs

```diff
@@ -2450,6 +2451,64 @@ public async Task When_Incremental_Load_ShouldStop()
 			);
 		}
 
+#if __IOS__ || __ANDROID__
+		[TestMethod]
+		public async Task When_Smooth_Scrolling()
+		{
+			// setup
+			var container = new Grid { Height = 210, VerticalAlignment = VerticalAlignment.Bottom };
+
+			var source = Enumerable.Range(0, 50).ToArray();
+			var lv = new ListView
+			{
+				ItemsSource = source,
+				ItemTemplate = FixedSizeItemTemplate, // height=29
+				ItemContainerStyle = BasicContainerStyle,
+			};
+			container.Children.Add(lv);
+
+			WindowHelper.WindowContent = container;
+			await WindowHelper.WaitForLoaded(lv);
+			await Task.Delay(1000);
+
+			// check the listview doesnt already have all items materialized
+			var count = lv.NativePanel?.GetChildren().Count();
+			Assert.IsTrue(count < source.Length, $"Native ListView is not {(count.HasValue ? $"virtualized (count={count})" : "loaded")}.");
+
+			// scroll to bottom
+#if __ANDROID__
+			lv.NativePanel.SmoothScrollToPosition(lv.Items.Count - 1);
+#elif __IOS__
+			if (lv.NativePanel is { } native)
+			{
+				// find last no-empty section to scroll
+				var path = Enumerable.Range(0, (int)native.NumberOfSections()).Reverse()
+					.Select(x => new { Section = x, Index = native.NumberOfItemsInSection(x) - 1 })
+					.Where(x => x.Index > 0)
+					.FirstOrDefault();
+				if (path != null)
+				{
+					native.ScrollToItem(NSIndexPath.FromRowSection(path.Index, path.Section), UICollectionViewScrollPosition.Bottom, animated: true);
+				}
+			}
```

Comment:
This should be exposed as part of the helper, as it is likely duplicated in the toolkit (or any part of code that wants to use this).

Reply:
So, after deliberation:
- The behavior is to change [`ListViewBase.ScrollIntoView`](https://docs.microsoft.com/en-us/uwp/api/windows.ui.xaml.controls.listviewbase.scrollintoview?view=winrt-22621) to be smooth scrolling by default on both iOS and Android.
- That behavior would be behind a feature flag to restore the original non-smooth behavior
- Add an uno-specific API that explicitly provides non-smooth `scrollintoview`

@Xiaoy312 @francoistanguay makes sense ?

Reply:
LVB.ScrollIntoView on windows doesnt animated the scroll, it just jump to there instantly.
I would argue that it would be best to keep that as is. And, expose the smooth scrolling as part of ListViewHelper.